ksqldb-engine/src/test/java/io/confluent/ksql/function/UdfLoaderTest.java

```diff
@@ -826,7 +826,7 @@ public void shouldInvokeUdafWithMethodWithNoArgs() throws Exception {
         "",
         "",
         "");
-    assertThat(creator.createFunction(AggregateFunctionInitArguments.EMPTY_ARGS),
+    assertThat(creator.createFunction(AggregateFunctionInitArguments.EMPTY_ARGS, Collections.EMPTY_LIST),
```

Comment:
Since this is a widening of the API, none of the existing tests depend on what is being passed into that function.

The function here is not public, so I'm unsure how much I'm worried with testing it. 

Also, I'm not sure what test would show things really well. 

That said, I did put together a rough example test which shows that the parameters are used.